### DISCUSSION TITLE
Allow using quotation marks to search for specific maps/creators etc

### DISCUSSION THREAD

**Discussion by RaiLTC** [ppy/osu#18062] [Ideas]
Currently the only way to search for something is using for example creator=name, but if we were to use that, song select will show anything that contains "name". For example, I was looking for creator=Rai, but I got results like Raiden, Kurai, KuraiPettan, Rai etc
Sorry for poor wording I'm quite tired

**Comment by RaiLTC** [comment]
Should apply for web and direct as well

**Reply by peppy** [reply]
Did you try this? It should already work.

**Reply by RaiLTC** [reply]
Yes, it currently does not work

**Reply by peppy** [reply]
On the website? What is the search criteria you are using?

**Reply by bdach** [reply]
See this one for example: https://osu.ppy.sh/beatmapsets?q=creator%3Drai

The discussion poster is looking to exact-match, and both client and web currently check whether the search term is contained in the creator field.

**Reply by RaiLTC** [reply]
This! Thank you very much for clarifying, I'm very bad at expressing what I want to say properly so I really appreciate it

As the title says, simply allow to use quotation marks to look for exact matches. For example if I were to look for myself (Rai), I would never find my own maps in direct or beatmap listing. Using creator="Rai" would find me instantly. This can apply for everything and not only creator

**Reply by peppy** [reply]
Thanks, tracking at https://github.com/ppy/osu-web/issues/8873
